logback-classic/src/main/java/ch/qos/logback/classic/disruptor/LogProducerTranslator.java

```diff
@@ -0,0 +1,29 @@
+package ch.qos.logback.classic.disruptor;
+
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.spi.AppenderAttachableImpl;
+
+import com.lmax.disruptor.EventTranslatorTwoArg;
+
+/**
+ * Producers Translator Object.
+ * 
+ * @author Liu-Luke
+ */
+
+public class LogProducerTranslator {
+
+	public LogProducerTranslator() {

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://www.codacy.com/app/tony19/logback/file/2395872185/issues/source?bid=3222618&fileBranchId=3222679#l16)